README.md

```diff
@@ -6,5 +6,6 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+Rails is a web application development framework.A web framework - a tool that helps us quickly and easily build web applications
 1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
 1. Describe the flow of a PATCH request as it goes through the app.
```

Comment:
Your path from user request through to DB looks mostly right (though the thing that's actually) but the server is missing from your diagram. Additionally, the view is not a standalone component that sends a response back to the client; rather, it is a passive tool that is used by the controller to construct the response.